ebu_tt_live/bindings/validation/timing.py

```diff
@@ -43,6 +43,13 @@ def _pre_init_variables(self, dataset, element_content):
         self._computed_end_time = None
         self._semantic_dataset = dataset
 
+    def _exclude_badly_timed_elements_filter(self, value, element):
```

Comment:
Could this function be short handed?
`return True if not (element.begin is not None and element.end is not None and element.begin >= element.end) False` or something like that